pkg/aks/components/CruAks.vue

```diff
@@ -983,17 +983,6 @@ export default defineComponent({
       }
     },
 
-    // these fields are used purely in UI, to track individual nodepool components
-    cleanPoolsForSave(): void {
```

Comment:
if this is not needed anymore, I think we should be able to remove its usage here:
https://github.com/rancher/dashboard/blob/7c53e5fcc5b00e7aacdd4e1627f2e5a45a9ae815/pkg/aks/components/CruAks.vue#L302

Reply:
You're right. Interesting that that didn't throw errors.